src/game/ServerList.cpp

```diff
@@ -1,40 +1,64 @@
 #include "ServerList.h"
 
-#include "utils/json.hpp"
+#include <optional>
```

Comment:
you don't need to change this, just recognizing a problem across the codebase: we need well defined style rules for the order of STL, third-party, and internal headers